src/_common/scroll/inview/container.ts

```diff
@@ -1,105 +1,73 @@
-import 'rxjs/add/operator/throttleTime';
-import { Scroll } from '../scroll.service';
-import { ScrollWatcher } from '../watcher.service';
 import { AppScrollInview } from './inview';
 
-/**
- * When the scroll velocity is below this minimum distance in px, we will check inview items every
- * ScrollThrottleTime. This ensures that if they're scrolling constantly, but really slowly, we'll be
- * able to do the checks even though debounce won't fire.
- */
-const DefaultScrollVelocityMinimum = 2000;
-const DefaultScrollThrottleTime = 300;
-
 export class ScrollInviewContainer {
-	items: AppScrollInview[] = [];
-
-	private lastScrollTop = 0;
-	private lastThrottleTime = Date.now();
-	private lastScrollHeight?: number;
-	private queueTimeout?: number;
-
-	get context() {
-		return this.scrollWatcher.context;
-	}
-
-	constructor(
-		private readonly scrollWatcher: ScrollWatcher,
-		private readonly throttle = DefaultScrollThrottleTime,
-		private readonly velocity = DefaultScrollVelocityMinimum
-	) {
-		if (GJ_IS_SSR) {
-			return;
-		}
```

Comment:
This doesn't matter since we only use this container class in `mounted` functions now. The code was drastically simplified from before to make it so we don't need any checks anymore.